lib/repl.js

```diff
@@ -732,10 +751,12 @@ function complete(line, callback) {
     }
   }
 
+  var self = this;
```

Comment:
Can you please use an arrow function so that you don't need the `var self = this` trick?